signer/src/ws_server/mod.rs

```diff
@@ -43,17 +46,48 @@ impl From<ws::Error> for ServerError {
 	}
 }
 
+/// Builder for `WebSockets` server
+pub struct ServerBuilder {
+	handler: Arc<IoHandler>,
+}
+
+impl Default for ServerBuilder {
```

Comment:
ah, i didn't realize that it was for clippy's benefit. I guess intuitively i would probably have it the other way -- have `new() `call `default()` rather than the other way around, but that's definitely just a personal preference.